### ISSUE TITLE
Where do I get the model F1 Score?

### ISSUE THREAD

**Issue by Walllace** [Franck-Dernoncourt/NeuroNER#174]
I uploaded the i2b2 2014 data set (that is, the 2 training sets and the 1 test set per the readme). After turning this to the BRAT format and putting the train, test, valid folders all into 1 data set, I've gone ahead and run the NeuroNer like so:

```
from neuroner import neuromodel
n2 = neuromodel.NeuroNER(train_model=False, use_pretrained_model=True, dataset_text_folder="./data/newdata", pretrained_model_folder ="./trained_models/i2b2_2014_glove_spacy_bioes")
```

The code managed to run without any errors and gave me the summary statistics, I just don't understand how I can obtain the model f1 score. Am I missing something?
